packages/components/src/Link/Link.stories.mdx

```diff
@@ -0,0 +1,69 @@
+import { ArgsTable, Canvas, Meta, Story } from "@storybook/addon-docs";
+import { Link } from ".";
+
+# Link
+
+<Meta title="Components/Link" component={Link} />
+
+```ts
+import { Link } from "@jobber/components/Link";
+```
+
+<Canvas>
+  <Story
+    name="Link"
+    args={{
+      text: "Link Component Notes",
```

Comment:
This prop doesn't match the component anymore, I think it can be removed.

Reply:
Ah yes, this vestigial prop has been surgically removed.  Good catch.

Reply:
🔪